VRDR/DeathRecord.cs

```diff
@@ -7270,9 +7270,13 @@ public string InjuryDescription
         {
             get
             {
-                if (InjuryIncidentObs?.Value != null)
+                if (InjuryIncidentObs?.Value != null && InjuryIncidentObs.Value as CodeableConcept != null)
```

Comment:
The `as` check makes the first null comparison redundant, so only one should be needed.
Perhaps instead of casting it twice (in the `if` with `as` and then again on line 7275), this should be something like

```c#
CodeableConcept concept = InjuryIncidentObs?.Value as CodeableConecpt;
if (concept != null)
{
...
}
```

Reply:
done